score/etl/sources/census/etl_utils.py

```diff
@@ -0,0 +1,47 @@
+from pathlib import Path
+import csv
+import os
+from config import settings
+
+from utils import remove_files_from_dir, remove_all_dirs_from_dir, unzip_file_from_url
+
+
+def reset_data_directories(data_path: Path) -> None:
+    census_data_path = data_path / "census"
+
+    # csv
+    csv_path = census_data_path / "csv"
+    remove_files_from_dir(csv_path, ".csv")
+
+    # geojson
+    geojson_path = census_data_path / "geojson"
+    remove_files_from_dir(geojson_path, ".json")
+
+    # shp
+    shp_path = census_data_path / "shp"
+    remove_all_dirs_from_dir(shp_path)
+
+
+def get_state_fips_codes(data_path: Path) -> list:
```

Comment:
Hmm, every time we run this we're going to have to download a file from remote? I'm wondering if we just want to store the list as a list rather than invoking an outbound request every time.

Reply:
Nope, this is stored on disk the first time it's run (see line 29 below, so it would be downloaded just once per container generation).